dotcom-rendering/src/components/ManyNewslettersForm.tsx

```diff
@@ -63,10 +62,21 @@ export const inputWrapperStyle = css`
 	}
 `;
 
-export const formAsideStyle = css`
+export const formAsideStyle = (hideOnMobile: boolean) => css`
+	overflow: hidden;
+	transition: max-height 1s;
+	max-height: 200px;
```

Comment:
Non-blocking - when would the height be less than `200`? And is it an issue that we're potentially cutting off privacy notice information?

Reply:
It *should* always be less than 200 - with the current text and the current formatting on a 300px viewport, the height if the aside is 96px. For the transition to work, max-height needs to have a numerical value - 200px is high enough that  it shouldn't cut off any off any of the information, but that assumes the message wont be changed to something much longer and the text size / line height won't be dramatically increased 🤔  - IE it isn't future-proof.

I could add a `useEffect` hook to set a state value for element's `scrollHeight` and pass the value as an argument to `formAsideStyle` - but I thought that might be too much code. 

Could also increase the maxHeight and the transition time to give some extra leeway - `300px` and `1.5s linear` looks okay. Making the maxHeight too high gives you an oddly-paced transition.